.github/workflows/build-ws-container.yml

```diff
@@ -6,14 +6,25 @@ on:
   # can be run manually on https://github.com/cockpit-project/cockpit/actions
   workflow_dispatch:
 
+env:
+  tag: ${{ vars.REGISTRY_REPO || 'quay.io/cockpit/ws' }}
```

Comment:
This is a bit confusing; "tag" sounds to me like ":latest". Maybe `repo` or `registry_repo` to match the vars?